uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Test/Fabricator/TOILRequest.php

```diff
@@ -2,29 +2,42 @@
 
 use CRM_HRLeaveAndAbsences_BAO_LeaveRequest as LeaveRequest;
 use CRM_HRLeaveAndAbsences_BAO_TOILRequest as TOILRequest;
+use CRM_HRLeaveAndAbsences_Test_Fabricator_LeaveBalanceChange as LeaveBalanceChangeFabricator;
 
 class CRM_HRLeaveAndAbsences_Test_Fabricator_TOILRequest {
 
   private static $leaveRequestStatuses;
   private static $toilAmounts;
 
-  public static function fabricate($params) {
+  public static function fabricate($params, $withBalanceChanges = false) {
```

Comment:
My impression was that almost (i would say something like 90%) of the tests required that you add the true param. But, I think that the argument in favor of consistency is a good one, so let's keep it as false